src/Workspaces/Core/Portable/Rename/Renamer.cs

```diff
@@ -23,6 +26,98 @@ public static class Renamer
                 newName, optionSet, cancellationToken);
         }
 
+        /// <summary>
```

Comment:
ths is public documentation, so let's make sure it's formatted well.  That means things like `<list>` and `<item>` not `*`.